images/iscsi-target-tgtd/run-tgt.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# https://fedoraproject.org/wiki/Scsi-target-utils_Quickstart_Guide
+
+set -e
+
+die() { echo "ERR: $@" ; exit 2 ; }
+
+SIZE_MB=${SIZE:-1024}
+WWN=iqn.2017-01.io.kubevirt:sn.42
+
+if [[ "$GENERATE_DEMO_OS_SEED" == "alpine" ]]; then
+  echo "Creating alpine demo OS image as requested"
+  mkdir -p /volume
+  curl -o /volume/file.img https://nl.alpinelinux.org/alpine/v3.5/releases/x86_64/alpine-virt-3.5.1-x86_64.iso
+elif [[ -n "$GENERATE_DEMO_OS_SEED" ]]; then
+  echo "Creating qemu demo OS image as requested"
+  mkdir -p /volume
+  curl http://download.qemu-project.org/linux-0.2.img.bz2 | bunzip2 > /volume/file.img
```

Comment:
Can we make this more uniform? I think the pod is much easier to understand and to handle, if you say: A pod restart resets everything, except if you use a persistent volume for `/volume`. Then all images are preserved.